PayPal.xcodeproj/project.pbxproj

```diff
@@ -1568,6 +1576,7 @@
 					"@executable_path/Frameworks",
 					"@loader_path/Frameworks",
 				);
+				MACOSX_DEPLOYMENT_TARGET = "$(RECOMMENDED_MACOSX_DEPLOYMENT_TARGET)";
```

Comment:
I am not really understanding these large diffs in pbxproj file. I did update to Xcode 14.3.1 last week but didn't see all these changes before. I was prompted one time in Xcode to make some recommended changes to file settings but I reverted those changes. I initially had a lot of problems with running unit tests but I resolved it by unchecking TestShared as target in ConfirmPaymentSourceRequest_Test